media-video/obs-studio/metadata.xml

```diff
@@ -12,6 +12,7 @@
     internally.
   </longdescription>
   <use>
+    <flag name="browser">Enable browser source support via CEF.</flag>
```

Comment:
I see this USE flag allows users to use a browser source in OBS. The ebuild downloads the browser source plugin as a binary -- which is understandable given how long it takes to compile the plugin from source -- but perhaps this USE flag should reflect that so users know what's happening. Would it make more sense to call this USE flag `browser-bin` or something similar to make it obvious that the plugin is downloaded as a binary?

Reply:
Possibly `browser-src-bin` might be a better name, but the description should definitely be updated to reflect this, as you rightly note! :smiley:

Reply:
The plugin itself is compiled from the git submodule. Only CEF is downloaded as a binary. I'll update the description but not sure about changing the flag's name.

Reply:
Right, and I've seen some talk about ebuilds only including '-bin' if there is a 'from source' option available, which as far as I know, we don't have have.

Of course this isn't about an ebuild name, and I don't remember if there was any sort of consensus reached though.

I don't mind if it's somehow mentioned in the description of the flag.

Reply:
I'm sorry, yes it's the CEF that is downloaded as a binary, so mentioning it in the description would be good. Since the plugin itself is compiled from source, I suppose `browser` is fine as a USE flag name. If in the future we decide to allow users to compile the CEF, then perhaps a `cef` USE flag would be added, so I reckon there's no naming conflict there between the USE flags.

Reply:
I've updated the description of the USE flag